Do not install implibs for modules (determined by install_path)

--- origsrc/midori-0.5.2/.waf-1.5.19-d046dea57dbefcabd071866ac51eb518/wafadmin/Tools/ccroot.py	2010-09-29 13:15:44.000000000 -0500
+++ src/midori-0.5.2/.waf-1.5.19-d046dea57dbefcabd071866ac51eb518/wafadmin/Tools/ccroot.py	2013-07-25 03:04:10.186115200 -0500
@@ -315,7 +315,8 @@ def apply_implib(self):
 	implib=self.env['implib_PATTERN']%os.path.split(self.target)[1]
 	implib=dll.parent.find_or_declare(implib)
 	self.link_task.outputs.append(implib)
-	self.bld.install_as('${LIBDIR}/%s'%implib.name,implib,self.env)
+	if self.install_path == os.path.abspath(self.env['LIBDIR'] + "/../bin"):
+		self.bld.install_as('${LIBDIR}/%s'%implib.name,implib,self.env)
 	self.env.append_value('LINKFLAGS',(self.env['IMPLIB_ST']%implib.bldpath(self.env)).split())
 def apply_vnum(self):
 	if not getattr(self,'vnum','')or not'cshlib'in self.features or os.name!='posix'or self.env.DEST_BINFMT not in('elf','mac-o'):
